client/packages/common/src/hooks/useDialog/useDialog.tsx

```diff
@@ -224,14 +236,30 @@ export const useDialog = (dialogProps?: DialogProps): DialogState => {
         sx={sx}
         TransitionComponent={Transition}
         disableEscapeKeyDown={false}
+        fullScreen={fullScreen}
       >
+        {fullScreen && (
+          <IconButton
+            icon={<CloseIcon />}
+            color="primary"
+            onClick={() => {
+              onClose && onClose();
+              hideDialog();
+            }}
+            sx={{ position: 'absolute', right: 0, top: 0, padding: 2 }}
+            label={t('button.close')}
+          />
+        )}
         {title ? <ModalTitle title={title} /> : null}
         <form
           style={{
             display: 'flex',
             flexDirection: 'column',
             flex: '1 1 auto',
             overflow: 'auto',
+            ...(fullWidthOnMobile
+              ? {}
+              : { width: dimensions.width, margin: '0 auto' }),
```

Comment:
Wide modals (like outbound shipment edit) should take up as much width as possible, but most want the content centered